redhat-cop/infra-ansible
# Enhancements to IdM based certs to align with LE roles/playbooks

### What does this PR do?
Updating the IdM cert role to align with the latest changes for Let's Encrypt role - mainly how the CSR role is used. 

### How should this be tested?
1. Make sure you have access to an IdM
2. Create an inventory based on the sample inventory 
3. Run the playbook to generate IdM certs

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
N/A

### People to notify
cc: @redhat-cop/infra-ansible @BinaryDevotee
